NameSurfer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * File: NameSurfer.java
+ * ---------------------
+ * When it is finished, this program will implements the viewer for
+ * the baby-name database described in the assignment handout.
+ */
+
+import acm.program.*;
+import java.awt.event.*;
+import javax.swing.*;
+
+public class NameSurfer extends Program implements NameSurferConstants {
+
+/* Instance Variables */
+	private JTextField tf;
+	private JButton graph;
+	private JButton clear;
```

Comment:
Should be called graphButton and clearButton to be clear what these are referring to. Otherwise it might make reader think you're referring to the graph UI.